Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -605,6 +604,26 @@ namespace mu2e {
       A6105SmallExtrusion->AddMaterial(findMaterialOrThrow("G4_Zn"), 0.001);
     }
 
+    // Aluminium alloy 6061
+    mat = uniqueMaterialOrThrow("A6061");
+    {
+      G4Material* Al6061 = new G4Material(mat.name, 2.70*CLHEP::g/CLHEP::cm3, 4);
+      Al6061->AddMaterial(findMaterialOrThrow("G4_Al"), 0.980);
+      Al6061->AddMaterial(findMaterialOrThrow("G4_Mg"), 0.010);
+      Al6061->AddMaterial(findMaterialOrThrow("G4_Si"), 0.006);
+      Al6061->AddMaterial(findMaterialOrThrow("G4_Fe"), 0.004);
+    }
+
+    // Aluminium alloy 7075
+    mat = uniqueMaterialOrThrow("A7075");
```

Comment:
Should be Al7075? (same for Al6061)

Reply:
I also get an error about components sum to 1 here

Reply:
good catch. I had only tested the reco side. Fixed with the latest push.